streams/cn.m3u

```diff
@@ -1447,3 +1447,43 @@ http://yslk.chinashadt.com:1635/live/stream:di1.stream/playlist.m3u8
 http://yslk.chinashadt.com:1635/live/stream:di2.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="",龙岩综合 (540p)
 http://stream.lytv.net.cn/2/sd/live.m3u8
+#EXTINF:-1 tvg-id="",Ankang TV-1 News [Geo-blocked]
+http://124.47.34.186/PLTV/88888888/224/3221225855/index.m3u8
+#EXTINF:-1 tvg-id="",Beijing Traffic Radio TV [Geo-blocked]
+http://123.56.24.28:1935/live/fm1039/96K/tzwj_video.m3u8
```

Comment:
IPRD : http://lhttp.qingting.fm/live/336/64k.mp3

Reply:
https://jwplay.hebyun.com.cn/live/hbystv/1500k/tzwj_video.m3u8 could be another live to add.

the correct FM could be https://brtv-radiolive.rbc.cn/alive/fm1039.m3u8

Reply:
The original website is down (from my side) so I don't think that works anymore.

Reply:
One of the (many) problems I'm having with this list is geo-blocking. Secondly, occassionally "odd" things are happening when I attempt to access some of these sites (e.g. weird pop-ups, ads), so what I'm going to be doing to keep this going is to tread lightly and accept reviewed recommendations automatically.